src/components/workspace/Tabs/Tabs.tsx

```diff
@@ -22,6 +53,9 @@ const Tabs: FC = () => {
     close(filePath);
   };
 
+  const onMenuItemClick = (info: MenuInfo, filePath: Tree['path']) => {
```

Comment:
This will force the rerender of the whole tab list every time `Tabs` component gets an update. Event handlers should be enclosed in `useCallback`.